### PR TITLE
remove all absolute links inside the doc

### PR DESCRIPTION
**Pull request by emilisev**

### CODE REVIEW

File: docs/EN/remote-control.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ Dragged into Google drive folder, and then added onto the follower phone by clic
 
 ### Synchronization- AAPSClient and AAPS set up (for Version 3.2.0.0 above)
 
-Once __AAPSClient__ apk is installed on the follower phone, the user must ensure their ‘Preferences’ in Config Builder are correctly set up and aligned with __AAPS__ for Nightscout 15 (see Release Notes [here](https://androidaps.readthedocs.io/en/latest/Installing-AndroidAPS/Update-to-new-version.html)). The example below provides Synchronization guidance for NSClient and NSClientV3 using Nightscout15 but there are other options available with __AAPS__ (e.g xDrip+). 
+Once __AAPSClient__ apk is installed on the follower phone, the user must ensure their ‘Preferences’ in Config Builder are correctly set up and aligned with __AAPS__ for Nightscout 15 (see Release Notes [here](Installing-AndroidAPS/Update-to-new-version.md)). The example below provides Synchronization guidance for NSClient and NSClientV3 using Nightscout15 but there are other options available with __AAPS__ (e.g xDrip+).
```
Comment (anbraten) [inline docs/EN/remote-control.md:323]:
```suggestion
Once __AAPSClient__ apk is installed on the follower phone, the user must ensure their ‘Preferences’ in Config Builder are correctly set up and aligned with __AAPS__ for Nightscout 15 (see Release Notes [here](./Installing-AndroidAPS/Update-to-new-version.md)). The example below provides Synchronization guidance for NSClient and NSClientV3 using Nightscout15 but there are other options available with __AAPS__ (e.g xDrip+). 
```

File: docs/EN/remote-control.md [hunk 2]
```diff
@@ -777,7 +777,7 @@ Making adjustments in both the **Galaxy wearable** app on the **AAPS** phone and
 #### Troubleshooting Sony smartwatch setup
 
 Although it was discontinued a few years ago, if you are using a Sony Smartwatch SW 3 please see here for a troubleshooting guide: 
-[Troubleshooting Sony Smartwatch SW 3](https://androidaps.readthedocs.io/en/latest/Usage/SonySW3.html)
+[Troubleshooting Sony Smartwatch SW 3](Usage/SonySW3.md)
```
Comment (anbraten) [inline docs/EN/remote-control.md:780]:
```suggestion
[Troubleshooting Sony Smartwatch SW 3](./Usage/SonySW3.md)
```

**Review by anbraten — APPROVED** [review]

File: docs/EN/remote-control.md (hunk 2 above)
Comment (emilisev) [reply docs/EN/remote-control.md:780]: It works with or without the initial ./

Comment (psonnera) [reply docs/EN/remote-control.md:780]: I prefer explicit relative links

**Pull request merged**